src/content/blog/arm64-apple-silicon-m1-ddev-local-what-does-it-all-mean.md

```diff
@@ -13,52 +13,40 @@ categories:
+Building DDEV for ARM64 was a monumental task, and a huge demonstration of the beauty of open source. Dennis has a Surface Pro X (ARM64) that he loves and wanted to be able to develop on… So he took on the task. It involved:
 
-Building DDEV-Local for ARM64 was a monumental task, and a huge demonstration of the beauty of open source. Dennis has a Surface Pro X (ARM64) that he loves and wanted to be able to develop on… So he took on the task. It involved:
-
-- Reworking all the DDEV-Local Docker images (ddev-webserver, ddev-dbserver, ddev-ssh-agent, ddev-router) so they could be built for ARM64.
-- Providing ARM64 builds of certain key upstream tools which were incorporated into Docker images or required alongside DDEV-Local, like mkcert, mailhog, and docker-gen.
+- Reworking all the DDEV Docker images (ddev-webserver, ddev-dbserver, ddev-ssh-agent, ddev-router) so they could be built for ARM64.
+- Providing ARM64 builds of certain key upstream tools which were incorporated into Docker images or required alongside DDEV, like mkcert, mailhog, and docker-gen.
 - Reworking the image build process to use Docker’s new [multi-architecture](https://www.docker.com/blog/tag/multi-architecture/) support, which allows a single image on [hub.docker.com](http://hub.docker.com) to have both AMD64 and ARM64 versions, and to pull the correct one as needed.
-- Building the DDEV-Local Go binary for AMD64.
-
-[Get the latest DDEV-Local here](https://github.com/ddev/ddev/releases)
-
-## When will DDEV-Local Work on Apple Silicon?
-
-People are just starting to receive the brand new macOS machines with Apple Silicon; we ordered ours and expect it to arrive in late November.
-
-However, Docker has the same problem as everybody else with getting Docker Desktop to run on Apple Silicon (see their [announcement](https://www.docker.com/blog/apple-silicon-m1-chips-and-docker/)). As of November, 2020, Docker Desktop doesn’t run yet on Apple Silicon, but they’re promising a technical preview soon, and they’re happy with their progress on it.
-
-Besides the dependency on Docker Desktop, the only other known issue in getting DDEV working on Apple Silicon is compiling the Go binary. The current version of Go, v1.15, does not support building Darwin/ARM64, the name of the architecture. However, the upcoming Go v1.16 ([due in February, 2021](https://github.com/golang/go/wiki/Go-Release-Cycle)) does. And it’s available a few ways right now. Experiments with building have been successful but, of course, we can’t run on real hardware until we get it.
+- Building the DDEV Go binary for AMD64.
 
-So the answer is… DDEV-Local will hopefully run on Apple Silicon Real Soon, probably just after a workable Docker Desktop is made available.
+**The good news for all of us is that DDEV is incredibly performant running natively on Apple Silicon.. and works great everywhere else too.**
```

Comment:
@rfay Would you mind accepting a trivial change here? I just revamped the build setup and Cloudflare *should* swoop in with a comment and automatic build here after the next commit. 🤞 

```suggestion
**The good news for all of us is that DDEV is incredibly performant running natively on Apple Silicon, and it works great everywhere else too.**
```

Reply:
Thanks for reading it!